package.json

```diff
@@ -29,6 +29,7 @@
   "devDependencies": {
     "code": "2.x.x",
     "hapi": "10.x.x",
+    "insync": "2.1.x",
```

Comment:
It doesn't have a method to pass an array of functions that execute in series. Also, per the items documentation 

> Use async for all your application needs.

items is for the very few flow control options hapi core needs, it is not a general purpose async library.